src/odemis/gui/model/__init__.py

```diff
@@ -1276,11 +1276,24 @@ def __init__(self, main):
         assert main.microscope is not None
         super(FastEMAcquisitionGUIData, self).__init__(main)
 
-        self.calibration_regions = model.VigilantAttribute({})  # dict, number --> FastEMROC
+        # calibration regions for calibration step 2
+        self.regions_calib_2 = model.VigilantAttribute({})  # dict, number --> FastEMROC
         for i in main.scintillator_positions:
-            self.calibration_regions.value[i] = fastem.FastEMROC(str(i), acqstream.UNDEFINED_ROI)
+            self.regions_calib_2.value[i] = fastem.FastEMROC(str(i), acqstream.UNDEFINED_ROI)
+
+        # calibration regions for calibration step 3
+        self.regions_calib_3 = model.VigilantAttribute({})  # dict, number --> FastEMROC
+        for i in main.scintillator_positions:
+            self.regions_calib_3.value[i] = fastem.FastEMROC(str(i), acqstream.UNDEFINED_ROI)
+
         self.projects = model.ListVA([])  # list of FastEMProject
 
+        self.is_calib_1 = model.BooleanVA(False)
+        self.is_calib_2 = model.BooleanVA(False)
+        self.is_calib_3 = model.BooleanVA(False)
```

Comment:
Could you add a comment explaining what True/False means? Is it running the calibration? Or is it done calibrating? Is it entirely calibrated for all the active scintillators?